plugins/main/public/controllers/management/components/management/cdblists/views/list-editor.tsx

```diff
@@ -404,6 +498,7 @@ class WzListEditor extends Component {
                       isDisabled={!addingKey}
                       fill
                       onClick={() => this.addItem()}
+                      disabled={this.state.isInvalid.length > 0}
```

Comment:
extract to a constant: `hasInvalidState`

Reply:
Done [310743b](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7171/commits/310743b1942f73af2d2e2df3efd88ba0be294136)